packages/nextra/src/loader.js

```diff
@@ -21,6 +21,17 @@ function getFileName (resourcePath) {
   return removeExtension(path.basename(resourcePath))
 }
 
+const parseJsonFile = (path) => {
+  let parsed
+  try {
+    parsed = JSON.parse(file)
+  } catch (err) {
+    throw Error(`Error parsing ${path}, make sure it's a valid JSON \n` + err)
+  }
+
+  return parsed
```

Comment:
Thank you! Will this error thrown make the dev server recoverable? Like do I need to restart the dev server once I fix the JSON file?

Reply:
Couldn't make the examples working with the local version so I can't test it manually. But I would assume so, since if there's any throw in a webpack loader it's a node process and it would crash. I believe there's a way to "accumulate" those errors in the webpack plugin API somewhere. 

It might be simpler to `console.error` thought.

Let me try to run examples locally.